actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -69,6 +69,10 @@ module ActionDispatch
   # size of the browser screen. These two options are not applicable for
   # headless drivers and will be silently ignored if passed.
   #
+  # There is support for running headless browsers, such as Headless Chrome
+  # and Headless Firefox. You can use them by putting +:headless_chrome+ or
+  # +:headless_firefox+ in the +:using+ argument.
+  #
```

Comment:
Thanks for adding this to the docs. I think it would read better as the following:

> Headless browsers such as headless Chrome and headless Firefox are also supported. You can use these browsers by setting the +:using+ argument to +:headless_chrome+ or +:headless_firefox+.